query-service-impl/src/test/resources/prometheus_request_handler.conf

```diff
@@ -0,0 +1,42 @@
+{
+  name = raw-service-prometheus-handler
+  type = prometheus
+  clientConfig = ""
+  requestHandlerInfo {
+    tenantAttributeName = tenant_id
+    prometheusViewDefinition {
+      viewName = rawServiceView
```

Comment:
can we also add?
scope = SERVICE

Reply:
added metricScope key, since attributes can be from other scope